novm/manager.py

```diff
@@ -356,8 +360,11 @@ def run_novmm(self,
             # Add extra (debugging) arguments.
             args.extend(["-%s" % x for x in vmmopt])
 
+            # Add log file.
```

Comment:
I'd like to keep the output if --nofork is specified.

How about something simple like this?

if not nofork:
   args.extend["-logfile=%s" % self._instances.file("%d.log" % os.gettpid())])